articles/python/azure-sdk-sovereign-domain.md

```diff
@@ -23,28 +23,32 @@ Pre-defined sovereign cloud constants are provided by the `azure_cloud` module o
 
 To use a definition, import the appropriate constant from `msrestazure.azure_cloud` and apply it when creating client objects. 
 
-When using `DefaultAzureCredential`, as shown in the following example, you also need to use the appropriate value from `azure.identity.AzureAuthorityHosts`.
+When using `DefaultAzureCredential`, as shown in the following example, you also need to use the appropriate value from `CLOUD.endpoints.active_directory`.
 
 ```python
 import os
-from msrestazure.azure_cloud import AZURE_CHINA_CLOUD as cloud
+from msrestazure.azure_cloud import AZURE_CHINA_CLOUD as CLOUD
 from azure.mgmt.resource import ResourceManagementClient, SubscriptionClient
 from azure.identity import DefaultAzureCredential
 
-# Assumes the subscription ID to use is in the AZURE_SUBSCRIPTION_ID environment variable
+# Assumes the subscription ID and tenant ID to use are in the AZURE_SUBSCRIPTION_ID,
+# AZURE_TENANT_ID environment variables
 subscription_id = os.environ["AZURE_SUBSCRIPTION_ID"]
+tenant_id = os.environ["AZURE_TENANT_ID"]
 
 # When using sovereign domains (that is, any cloud other than AZURE_PUBLIC_CLOUD),
 # you must use an authority with DefaultAzureCredential.
-credential = DefaultAzureCredential(authority=cloud.endpoints.active_directory)
+credential = DefaultAzureCredential(authority=CLOUD.endpoints.active_directory, tenant_id=tenant_id)
```

Comment:
`tenant_id` must be provided, otherwise the token will be issued to the `common` tenant (home tenant), causing failure on subscriptions from other tenants.